runtime/integration-tests/pendulum/src/tests.rs

```diff
@@ -0,0 +1,350 @@
+use crate::{polkadot_test_net::*, setup::*};
+use frame_support::{
+	assert_ok,
+	traits::{fungible::Mutate, fungibles::Inspect, Currency},
+};
+use pendulum_runtime::{Balances, PendulumCurrencyId, RuntimeOrigin, Tokens, XTokens};
+use sp_runtime::{traits::AccountIdConversion, MultiAddress};
+use xcm::latest::{Junction, Junction::*, Junctions::*, MultiLocation, NetworkId, WeightLimit};
+use xcm_emulator::{Junctions, TestExt};
+
+use polkadot_core_primitives::{AccountId, Balance};
+use polkadot_parachain::primitives::Sibling;
+
+const DOT_FEE: Balance = 3200000000;
+const ASSET_ID: u32 = 1984; //Real USDT Asset ID from Statemint
+const INCORRECT_ASSET_ID: u32 = 0;
+pub const UNIT: Balance = 1_000_000_000_000;
+pub const TEN: Balance = 10_000_000_000_000;
+const FEE: u128 = 421434140;
```

Comment:
Same here, what is this fee value about? Does it have to be this value? Please add a comment above that explains its usage